cmd/minishift/cmd/config/config.go

```diff
@@ -77,6 +77,9 @@ var (
 	OpenshiftEnv      = createFlag("openshift-env", nil, nil, nil, false)
 	Metrics           = createFlag("metrics", SetBool, nil, nil, true)
 	Logging           = createFlag("logging", SetBool, nil, nil, true)
+	// future enabled flags
+	ServiceCatalog      = createFlag("service-catalog", SetBool, nil, nil, true)
+	ExtraClusterUpFlags = createFlag("extra-clusterup-flags", SetString, nil, nil, true)
```

Comment:
As discussed on the issue, instead of adding more flags to the command line, I'd prefer to do the 'extra-clusterup-flags' as a environment variable. A power user backdoor so to say. It allows to run experiments and test new options early without having to make changes to the code.

Regarding, _service-catalog_, I am against adding this option until we can confirm this works. Once it is working in a consistent way, we can expose it as done here.